zulipterminal/ui_tools/boxes.py

```diff
@@ -36,8 +37,14 @@ def main_view(self, new: bool) -> Any:
     def set_editor_mode(self) -> None:
         self.view.controller.enter_editor_mode_with(self)
 
-    def private_box_view(self, button: Any=None, email: str='') -> None:
+    def private_box_view(self, button: Any=None, email: str='',
+                         subscriber_ids: List[int]=[]) -> None:
```

Comment:
OK, a shallow copy is causing the crash here rather than the change in recipients. I think it is fine to not have the current user stored in the recipients at all? That would also be consistent with the other recipient data we're storing in `MessageBox`

Reply:
Updating subscribers requires a little more discussion, opened #737

Reply:
On second thought, this wouldn't be too hard I'll fix this in the same PR